_pages/home/updating-to-version-2211.35/modernize-apps-migrated-from-6.8-to-2211.19.md

```diff
@@ -1,466 +1,473 @@
 
-### `angular.json`
+1. In the `architect > build` section of `angular.json`, change the value `"builder": "@angular-devkit/build-angular:browser",`
+   to `"builder": "@angular-devkit/build-angular:application",`.
 
-1. In the section `architect > build` please change the value `"builder": "@angular-devkit/build-angular:browser",`
-   to `"builder": "@angular-devkit/build-angular:application",`
+   The updated section should appear as follows:
 
-```diff
+```json
         "architect": {
           "build": {
--           "builder": "@angular-devkit/build-angular:browser",
-+           "builder": "@angular-devkit/build-angular:application",
+            "builder": "@angular-devkit/build-angular:application",
 ```
 
-Why: we're configuring here the new `application` builder for Angular v17 and later, which is the recommended fast and future-proof builder for Angular v17 and later.
+   This step configures the new `application` builder for Angular 17 and newer. This is the recommended, faster, and "future proof" builder for Angular version 17 and newer.
 
-1. In the section `architect > build > options` please apply the all the following modifications, to adapt to the new configuration format for the new builder:
+1. In the `architect > build > options` section of `angular.json`, apply all of the following modifications, to adapt to the new configuration format for the new builder:
 
-2.1 rename the property `"main"` to `"browser"`
+   1. Rename `"main": "src/main.ts"` to `"browser": "src/main.ts"`.
 
-```diff
-        "architect": {
-          "build": {
-           "options": {
--             "main": "src/main.ts",
-+             "browser": "src/main.ts",
-```
+      The updated section should appear as follows:
 
+      ```json
+      "architect": {
+        "build": {
+         "options": {
+            "browser": "src/main.ts",
+      ```
 
-2.2. In the section `architect > build > configurations > development` please remove 3 properties: `"buildOptimizer"`, `"vendorChunk"`, `"namedChunks"`
+   1. In the `architect > build > configurations > development` section, remove the following three properties: `"buildOptimizer"`, `"vendorChunk"`, and `"namedChunks"`
 
-```diff
-        "architect": {
-          "build": {
-           "configurations": {
-             "development": {
--              "buildOptimizer": false,
--              "vendorChunk": true,
--              "namedChunks": true
-```
+1. In the `"compilerOptions"` section of `tsconfig.json`, remove the following properties: `"baseUrl"`, `"forceConsistentCasingInFileNames"`, and `"downlevelIteration"`.
+1. In the same section, add the following properties: `"skipLibCheck": true` and `"esModuleInterop": true`.
 
-### `tsconfig.json`
+   The updated section should appear as follows:
 
-In the `"compilerOptions"` section, please:
+   ```json
+      "compilerOptions": {
+        "skipLibCheck": true,
+        "esModuleInterop": true,
+   },
```

Comment:
Let's remove the closing bracket or add /*...*/ - because those options are not meant to be alone there. There are plenty of other unchanged options that should remain there
```suggestion
```